src/esp/bindings/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ target_link_libraries(habitat_sim_bindings
 )
 
 if(WITH_GL_TENSOR)
-  target_compile_definitions(habitat_sim_bindings PRIVATE __ESP_WITH_GL_TENSOR__)
+  target_compile_definitions(habitat_sim_bindings PRIVATE __esp_with_gl_tensor__)
```

Comment:
Eh, sorry, should have mentioned that double underscores *anywhere* are reserved too :sweat_smile: [Specs.](https://en.cppreference.com/w/cpp/language/identifiers)